de/model.jl

```diff
@@ -97,31 +91,33 @@ function sol_matrix(pIn)
     return sol
 end
 
-" Cost function. Returns SSE + sum(abs(w)) between model and experimental RNAseq data. "
+" Single simulation cost function. "
+function sim_cost(pIn, exp_data)
+    return norm(solveODE(pIn) .- exp_data)
+end
+
+" Returns SSE between model and experimental RNAseq data. "
 function cost(pIn, exp_data)
-    neg = solveODE(pIn)
-    sse = norm(neg .- exp_data[:, 84])
+    return norm(sol_matrix(pIn) - exp_data)
+end
 
-    for i = 1:83
-        sol_temp = simKO(pIn, i)
-        sse += norm(sol_temp .- exp_data[:, i])
-    end
+" Cost function gradient. Returns SSE between model and experimental RNAseq data. "
```

Comment:
Note that this is the gradient of the cost function. Made a cost function right above this.